CMakeLists.txt

```diff
@@ -28,10 +28,15 @@ include(DiskImage)
 include(AddPackage)
 include(AutoModuleOption)
 
+if(NOT ASSETS_TYPE)
```

Comment:
assets should be an `option` shouldn't these? we actually have quite a few options, please see here:
doc/ProjectConfig.md